pkg/namespacestore/namespacestore.go

```diff
@@ -425,6 +425,10 @@ func RunCreateAzureBlob(cmd *cobra.Command, args []string) {
 
 // RunCreateNSFS runs a CLI command
 func RunCreateNSFS(cmd *cobra.Command, args []string) {
+	log := util.Logger()
+	if len(args[0]) > 63 {
```

Comment:
63 is not the correct limit here, the mount path is "/nsfs/" + namespacestore name

Reply:
Done.